easybuild/easyconfigs/s/sparsehash/sparsehash-2.0.3-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'sparsehash'
+version = '2.0.3'
+
+homepage = 'https://code.google.com/p/sparsehash/'
```

Comment:
should be `https://github.com/sparsehash/sparsehash`

Reply:
@JackPerdue ping?

Reply:
[j-perdue@terra2 2017b]$ grep ^source sparsehash-2.0.3-GCCcore-6.4.0.eb 
source_urls = ['https://github.com/sparsehash/sparsehash/archive']
sources = [SOURCELOWER_TAR_GZ]
[j-perdue@terra2 2017b]$ eb  --update-pr=4974 --pr-commit-msg="fix sourceurl"   ./sparsehash-2.0.3-GCCcore-6.4.0.eb 
== temporary log file in case of crash /tmp/j-perdue/tmp/eb-vA4DIu/easybuild-23IeRM.log
== resolving dependencies ...
== Determined branch name corresponding to easybuilders/easybuild-easyconfigs PR #4974: 20170803152041_new_pr_sparsehash203
== fetching branch '20170803152041_new_pr_sparsehash203' from https://github.com/JackPerdue/easybuild-easyconfigs.git...
== copying easyconfigs to /tmp/j-perdue/tmp/eb-vA4DIu/git-working-dirpQ3ZHL/easybuild-easyconfigs...
ERROR: No changed files found when comparing to current develop branch. Refused to make empty pull request.
[j-perdue@terra2 2017b]$ 


??????????????????

Reply:
@JackPerdue It's `homepage` that needs fixing, not `source_urls`, `eb` is probably right to say you didn't make any changes... :)